structs.go

```diff
@@ -298,13 +306,106 @@ type Channel struct {
 
 	// ApplicationID of the DM creator Zeroed if guild channel or not a bot user
 	ApplicationID string `json:"application_id"`
+
+	// Voice region ID for the voice channel, automatic when set to null
+	RTCRegion *string `json:"rtc_region"`
```

Comment:
It looks like this field is replacing `Region`, so we should probably remove that one while we're at it.